roles/openshift_node_group/templates/node-config.yaml.j2

```diff
@@ -50,7 +50,10 @@ kubeletArguments:
   cloud-provider:
   - {{ openshift_node_group_cloud_provider }}
 {% endif %}
-  node-labels: []
+  node-labels:
```

Comment:
@sdodson @mtnbikenc 

These don't appear to have any effect to actually label the node.